config/database.yml

```diff
@@ -1,6 +1,6 @@
 development:
   adapter: postgresql
-  url: <%= Rails.application.credentials.database_url %>
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
Generally it is a good practice to use `#fetch` to get essential environment variables, to make it "fail early". Otherwise lacking variable may cause unexpected behavior later in the run time.

For instance, GitHub Actions don't propagate "secrets" to forks, thus `DATABASE_URL` won't be available during tests run, causing a cryptic error message:

```
rails aborted!
ActiveSupport::MessageEncryptor::InvalidMessage: Cannot load database configuration:
ActiveSupport::MessageEncryptor::InvalidMessage
(erb):7:in `<main>'
/home/runner/work/feeder/feeder/config/environment.rb:5:in `<top (required)>'
/home/runner/work/feeder/feeder/bin/rails:9:in `require'
/home/runner/work/feeder/feeder/bin/rails:9:in `<top (required)>'
/home/runner/work/feeder/feeder/bin/spring:13:in `require'
/home/runner/work/feeder/feeder/bin/spring:13:in `<top (required)>'
bin/rails:3:in `load'
bin/rails:3:in `<main>'

Caused by:
ArgumentError: key must be 16 bytes
(erb):7:in `<main>'
/home/runner/work/feeder/feeder/config/environment.rb:5:in `<top (required)>'
/home/runner/work/feeder/feeder/bin/rails:9:in `require'
/home/runner/work/feeder/feeder/bin/rails:9:in `<top (required)>'
/home/runner/work/feeder/feeder/bin/spring:13:in `require'
/home/runner/work/feeder/feeder/bin/spring:13:in `<top (required)>'
bin/rails:3:in `load'
bin/rails:3:in `<main>'
Tasks: TOP => db:create => db:load_config => environment
(See full trace by running task with --trace)
```

In contrast `#fetch` would throw explicit `KeyError` which is easier to understand.

Reply:
Okay, but still why is GitHub Actions checking the sanity of your rails app in dev mode and not in test mode/ test environment

Reply:
This project has an environment variable with the master key used to decipher `Rails.application.credentials`. It works only for Actions running for the primary repository, not for forks.

I guess Rails Credentials silently fail to propagate `DATABASE_URL` to this config file. So it is actually not related to what I said above regarding `#fetch` method :)